BUILDING.md

```diff
@@ -45,6 +45,19 @@ Check that `cargo` is correctly installed by running
 cargo --version
 ```
 
+Next, install the nightly version of rustfmt
+```
+rustup toolchain install nightly
```

Comment:
```suggestion
rustup toolchain install $(< RUSTFMT_VERSION)
```
We don't use just any old nightly since `rustfmt` isn't always available in every one we bump this in a controlled manner as part of https://github.com/habitat-sh/habitat/blob/master/RELEASE.md#update-rustfmt.

Similarly for the `nightly` references in the later commands

Reply:
Won't that only work in the root Habitat directory?

Reply:
Well...if it's for installing, that's probably fine.  I'll add in a note to that effect.